spice_ev/generate/generate_schedule.py

```diff
@@ -882,18 +694,20 @@ def distribute_energy_balanced(period, is_charge_period, energy_needed, priority
         cur_time = s.start_time - s.interval
         for t in range(s.n_intervals):
             cur_time += s.interval
+            # charging window: curtailment present or res. load negative
+            charging_window = (curtailment[i] > EPS) or (residual_load[t] < -EPS)
             values = [
-                cur_time.isoformat(),      # timestamp
-                round(schedule[t], 3),     # schedule rounded to Watts
-                int(priorities[t] <= 2),   # charging window?
+                cur_time.isoformat(),  # timestamp
+                aggressive_round(schedule[t], 3),  # schedule rounded to Watts
+                int(charging_window),
                 round(original_residual_load[t], 3),
                 round(original_curtailment[t], 3),
                 round(residual_load[t], 3),
                 round(curtailment[t], 3),
```

Comment:
This was the only conflict, that could not be resolved by auto-merge @TabeaMaria